go.mod

```diff
@@ -3,7 +3,9 @@ module sigs.k8s.io/kubebuilder
 go 1.13
 
 require (
+	github.com/blang/semver v3.5.1+incompatible
 	github.com/gobuffalo/flect v0.2.1
+	github.com/golang/protobuf v1.3.4 // indirect
```

Comment:
Why it was added here? I think it is not required. See: https://github.com/kubernetes-sigs/kubebuilder/pull/1454

Reply:
This can probably be removed.